### PR TITLE
fix(catalog-backend): update the mock-data script to point to new example entities

### PR DESCRIPTION
**Pull request by freben**

### CODE REVIEW

**Comment by hooloovooo** [comment]
Nice catch

**Review by hooloovooo — APPROVED** [review]

File: plugins/catalog-backend/scripts/mock-data [hunk 1]
```diff
@@ -5,5 +5,5 @@ curl \
 --header 'Content-Type: application/json' \
 --data-raw '{
     "type": "github",
-    "target": "https://github.com/spotify/backstage/blob/master/plugins/catalog-backend/fixtures/two_components.yaml"
+    "target": "https://github.com/spotify/backstage/blob/master/plugins/catalog-backend/fixtures/example-components.yaml"
```
Comment (shmidt-i) [inline plugins/catalog-backend/scripts/mock-data:8]: Should it be `examples/example-components.yaml` instead? Feeling like fixtures is more test-oriented name

Comment (freben) [reply plugins/catalog-backend/scripts/mock-data:8]: sure, i can change that

**Comment by freben** [comment]
@shmidt-i @hooloovooo i changed that and also tweaked the process and READMEs a bit, take another look plz

**Review by shmidt-i — APPROVED** [review]

**Pull request merged**